client-common/src/main/java/org/apache/cassandra/sidecar/common/request/data/CreateRestoreJobRequestPayload.java

```diff
@@ -101,6 +105,9 @@ public CreateRestoreJobRequestPayload(@JsonProperty(JOB_ID) UUID jobId,
                              : importOptions;
         this.expireAtInMillis = expireAtInMillis;
         this.consistencyConfig = ConsistencyConfig.parseString(consistencyLevel, localDatacenter);
+        Preconditions.checkArgument(!localDatacenterOnly || StringUtils.isNotEmpty(localDatacenter),
+                                    "Must specify a non-empty localDatacenter when restoreToLocalDatacenterOnly is configured to true");
```

Comment:
I think the `non-empty` part is implied
```suggestion
                                    "Must specify a localDatacenter when restoreToLocalDatacenterOnly is true");
```

Reply:
ok. done